outdoorsmanship.lic

```diff
@@ -60,6 +60,7 @@ class Outdoorsmanship
 
     bput('release spell', 'You let your concentration lapse', "You aren't preparing a spell")
     bput('release mana', 'You release all', "You aren't harnessing any mana")
+    bput('release symb', "But you haven't", 'You release', 'Repeat this command')
```

Comment:
❌ This should not be done. You may need to merge in master, but I thought there was a comment explaining it.